spec/takeaway_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  it { is_expected.to be_instance_of Takeaway }
+
+  it "should contain a list of dishes with prices" do
+    expect(Takeaway::MENU).to be_instance_of Hash
+  end
+
+  it "should allow the user to place an order" do
```

Comment:
It doesn't test this, it just tests whether a method called 'start_order' exists.